metcdv3/coordinator.go

```diff
@@ -447,15 +445,11 @@ func (ec *EtcdV3Coordinator) Init(cordCtx metafora.CoordinatorContext) error {
 						return
 					default:
 					}
-					select {
-					case ec.failure <- ErrKeepAliveClosedUnexpectedly:
-						// Testing hook.
-						if stats != nil {
-							stats.failure++
-						}
-						metafora.Infof("metafora etcdv3 coordinator: %v: keep alive closed unexpectedly", ec.name)
-					default:
+					// Testing hook.
+					if stats != nil {
+						stats.failure++
```

Comment:
[question] do we need to return in the case of a test hook call?

Reply:
Since these tests haven't existed for a while I'm going to make an issue and assume that the caller is responsible for catching the panic in the test for now